### ISSUE TITLE
Addition of Googlebot profile to circumnavigate internet paywalls

### ISSUE THREAD

**Issue by sueridgepipe** [dillbyrne/random-agent-spoofer#369]
Details on accessing web content behind paywalls...

http://www.ghacks.net/2016/02/26/read-articles-behind-paywalls-by-masquerading-as-googlebot/

It references two addons, RefControl and User Agent Switcher, but I think this would be an easy addition into RAS with a selectable googlebot profile.

Thoughts?

**Comment by RoxKilly** [comment]
Interesting.  The capabilities discussed (spoofing referer and user agent) are close to what RAS already does, but I don't think developing tools for the purpose of circumventing paywalls fits the RAS mission.  RAS' spoofing powers are meant to defeat tracking; this seems out of scope to me. @dillbyrne  what do you think

**Comment by berrythesoftwarecodeprogrammar** [comment]
i think all they are asking for is for the useragent profile of googlebot to be added so one can use it when required

**Comment by forkoz** [comment]
Yes... we should have other crawler and bot profiles and how about adding custom profiles too.

**Comment by agentxray** [comment]
Definitely! This is in scope. Any common profile would add to our anonymity, and bots are quite common. I always enjoyed the bot useragents in Secret Agent. They could possibly be used for circumventing things other than paywalls too.  Custom profiles would also be good to have.
